src/main/java/gnu/io/NativeResource.java

```diff
@@ -139,7 +139,21 @@ private void loadResource(File resource) {
 		if(!resource.canRead())
 			throw new RuntimeException("Cant open JNI file: "+resource.getAbsolutePath());
 		//System.out.println("Loading: "+resource.getAbsolutePath());
+		try {
+			System.out.println("Going to load " + resource + ". Hit return...");
```

Comment:
Sorry, of course I wanted to submit only the change of line 43. I had forgotten about the extra debug code still being there and it wasn't in my editor windows (those are always too small) so I overlooked it when committing.